laser_mapper/launch/kitti/kitti_loop_closure.yaml

```diff
@@ -0,0 +1,115 @@
+LaserMapper: {
+  distance_to_consider_fixed: 80,
+  separate_distant_map: true,
+  create_filtered_map: true,
+  minimum_distance_to_add_pose: 1,
+  voxel_size_m: 0.10,
+  minimum_point_number_per_voxel: 1,
+  clear_local_map_after_loop_closure: true,
+  
+  world_frame: "map",
+  odom_frame: "world", # In the KITTI bag, transform is given from /world to /velodyne
+  sensor_frame: "velodyne",
+  
+  assembled_cloud_sub_topic: "/velodyne_points",
+  trajectory_pub_topic: "trajectory",
+  odometry_trajectory_pub_topic: "odometry_trajectory",
+  full_map_pub_topic: "full_map",
+  local_map_pub_topic: "local_map",
+  distant_map_pub_topic: "distant_map",
+  covariances_pub_topic: "covariances",
+
+  publish_local_map: true,
+  publish_distant_map: false,
+  publish_full_map: false,
+  map_publication_rate_hz: 3,
+  
+  publish_world_to_odom: true,
+  tf_publication_rate_hz: 10,
+  
+  # SegMatchWorker parameters
+  SegMatchWorker: {
+  localize: false,
+  close_loops: true,
+  
+  distance_between_segmentations_m: 10.0,
+
+  distance_to_lower_target_cloud_for_viz_m: 15.0,
+  align_target_map_on_first_loop_closure: false,
+  
+  ratio_of_points_to_keep_when_publishing: 1.0,
+
+  SegMatch: {
+    segmentation_radius_m: 60.0,
+    
+    filter_boundary_segments: true,
+    boundary_radius_m: 59.0,
+    filter_duplicate_segments: true,
+    centroid_distance_threshold_m: 2.0,
+
+    Segmenters: {
+      segmenter_type: "EuclideanSegmenter", #RegionGrowingSegmenter
+
+      rg_min_cluster_size: 100,
+      rg_max_cluster_size: 50000,
+      rg_knn_for_normals: 100,
+      rg_knn_for_growing: 30,
+      rg_smoothness_threshold_deg: 4.0,
+      rg_curvature_threshold: 0.05,
+      rg_radius_for_normals: 0.5,
+      
+      ec_tolerance: 0.2,
+      ec_max_cluster_size: 15000,
+      ec_min_cluster_size: 200
+    },
+
+    Descriptors: {
+      descriptor_types: ["EigenvalueBased"], # "EnsembleShapeFunctions"
+    },
+
+    Classifier: {
+     threshold_to_accept_match: 0.70,
+     n_nearest_neighbours: 200,
+     knn_feature_dim: 7,
```

Comment:
When we experimented with different descriptors that was used to set the feature to use for knn, in that case eigen values based.